src/WSCryptDecryptSoapClient.php

```diff
@@ -33,7 +33,6 @@ class WSCryptDecryptSoapClient
     public function __construct($testEnv = false, $caFile = null)
     {
         $soapClientDefaultOption = array(
-            'user_agent' => 'EndelWar-GestPayWS/1.3 (+https://github.com/endelwar/GestPayWS)',
```

Comment:
Why removing user agent from default soap client options and moving it to the bottom?